### PR TITLE
Update _objects.md

### PR DESCRIPTION
**Pull request by camnwalter**
I know parts of this are really outdated, so I removed a few things that you might want added back in.
I left the LookingAt section since the next few parts relied on information in it.

### CODE REVIEW

**Comment by camnwalter** [comment]
I was looking back over this and saw that most of the Inventory methods at the bottom just didn't exist, or they got moved to the Item class.

edit: Going to change the LookingAt and Inventory sections a lot

File: source/includes/_objects.md [hunk 1]
```diff
@@ -79,20 +80,41 @@ This is a very simple operation which just opens the book. You can also specify
...
 
-> This is how you can create a Display object
+> This is how you can create a `Display` object
 
 ```javascript
 var display = new Display();
 ```
 
-This display object is now created, but it doesn't do much of anything yet.
+This `Display` object is now instantiated, but it doesn't do much of anything yet.
```
Comment (mattco98) [inline source/includes/_objects.md]: I like "created" here better, as it's a bit simpler (i.e. more novice-friendly)

File: source/includes/_objects.md [hunk 2]
```diff
@@ -268,6 +300,34 @@ gui.close();
 
 These very simple methods open and close the gui, and neither take any arguments.
 
+# Inventory
+
+The Inventory object contains methods used for getting information about the user's inventory. It can be called
+through `Player.getInventory()`. More about the `Player` class can be found [later](#player).
+
+<aside class="warning">Looping through the inventory can be an intensive process if done rapidly.</aside>
```
Comment (mattco98) [inline source/includes/_objects.md]: Why is this warning needed? I haven't used the inventory API in a while (or maybe every), but at a glance it doesn't seem more intensive than anything else.

File: source/includes/_objects.md [hunk 3]
```diff
@@ -327,150 +387,103 @@ if you are holding down the key.
...
+In this case, we get the held item of the player. If the item isn't air, then it finds how damaged it is. If
+the durability isn't a number, that means the item has to be a block. Then, if the player isn't holding anything, it
+runs the last piece of the code, which is when the hand is empty.
+If you want to get the player's inventory as a list of `Items`, use `Player.getInventory()`.
 
 # XMLHttpRequests
 
-The XMLHttpRequests object has been removed in favor of the simpler, promise like [request](https://www.chattriggers.com/modules/v/request) module, its documentation can be found once downloaded.
\ No newline at end of file
+The XMLHttpRequests object has been removed in favor of the simpler, promise like [request](https://www.chattriggers.com/modules/v/request) module, its documentation can be found once downloaded.
```
Comment (mattco98) [inline source/includes/_objects.md]: Let's just remove this section all together as this was removed a while ago. If someone needs to know how to do http requests, they can ask in Discord. It might also be a good idea to have a section of the slate dedicated to common operations like http requests, but that would be for another PR.

**Review by mattco98 — CHANGES_REQUESTED** [review]
Make sure to squash your commits :smile:

**Comment by camnwalter** [comment]
> Make sure to squash your commits 😄

Um how do you do that? I'm a big noob at this. I'll do your suggestions later today though.
